01week/rockPaperScissors.js

```diff
@@ -8,12 +8,39 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
+function rockPaperScissors(a, b) {
 
-  // Write code here
+    // Write code here
+    var hand1 = a.toLowerCase().trim();
+    var hand2 = b.toLowerCase().trim();
 
+if (hand1 === hand2)
+return "It's a tie!"
+
+else if ((hand1 == "rock" && hand2 =="scissors") || (hand1 == "scissors" && hand2 == "rock")){
+  if(hand1=="rock")
+  return "Hand one wins!"
+  else 
+  return "Hand two wins!"
+}
+
+else if ((hand1 == "rock" && hand2 =="paper") || (hand1 == "paper" && hand2 == "rock")){
+  if(hand1=="paper")
+  return "Hand one wins!"
+  else 
+  return "Hand two wins!"
+  }
+
+  else if ((hand1 == "paper" && hand2 =="scissors") || (hand1 == "scissors" && hand2 == "paper")){
+    if(hand1=="scissors")
+    return "Hand one wins!"
+    else 
+    return "Hand two wins!"
+  }
 }
 
```

Comment:
This is good. I like the way you organized it. Another way you could have done this is find all the ways in which Hand 1 wins in one condition (else if) and else all other possibilities the other hand wins. It would keep you from having nested if statements, less conditions, and overall less lines of code.